types/mui-datatables/index.d.ts

```diff
@@ -58,24 +57,24 @@ interface MUIDataTableTextLabelsBody {
 }
 
 interface MUIDataTableTextLabelsPagination {
+    displayRows: string;
     next: string;
     previous: string;
     rowsPerPage: string;
-    displayRows: string;
 }
 
 interface MUIDataTableTextLabelsToolbar {
-    search: string;
     downloadCsv: string;
+    filterTable: string;
     print: string;
+    search: string;
     viewColumns: string;
-    filterTable: string;
```

Comment:
Why is this re-ordered?

Reply:
Alphabetized properties inside interface